### ISSUE TITLE
Create data container hotkey

### ISSUE THREAD

**Issue by antonkomarev** [secusu/web-app#17]
For desktop users `CTRL + ENTER` could create data container.

**Comment by orangecoloured** [comment]
I don't agree on this. This is not a chat and such UX can be not very user friendly.
I for example never expect forms submitted by Ctr+Enter.
